osidb/models.py

```diff
@@ -1157,6 +1158,39 @@ def delegated_resolution(self):
 
         return Affect.AffectFix.AFFECTED
 
+    def _validate_wontreport_products(self):
+        """
+        Validate that affected/wontreport only can be used for
+        to products associated with services
+        """
+        if (
+            self.affectedness == Affect.AffectAffectedness.AFFECTED
```

Comment:
I would not check affectedness here at all. This should be really just about WONTREPORT resolution alone.

Reply:
submitted a fixture.